src/components/InstallTabs/LinuxPanel/index.tsx

```diff
@@ -30,7 +30,7 @@ g++ libgcc linux-headers grep util-linux binutils findutils"
       {/* TODO when the new docs page is ready link to that page.  */}
       <a
         className="install__docs-button"
-        href="https://nodejs.org/en/download/package-manager/#nvm"
+        href="https://nodejs.dev/en/download/package-manager/#nvm"
```

Comment:
We don't have the `en` on the site as of now.


```suggestion
        href="https://nodejs.dev/download/package-manager/#nvm"
```